idea_town/frontend/static-src/app/lib/page-manager.js

```diff
@@ -37,6 +41,16 @@ export default class PageManager {
       const Ctor = this.pages[page];
       const newPage = new Ctor(opts);
 
+      if (newPage.headerScroll) {
```

Comment:
FWIW, I'd still unify `ScrollHeaderView` and `HeaderView` and have that single component handle that scrolling logic internally. The [built-in `prepareView` supplied by default subview registration](https://github.com/AmpersandJS/ampersand-view/blob/master/ampersand-view.js#L290) passes a reference to the parent view, so that component could check for something like `this.parent.headerScroll` and change based on the context where it's used.